### PR TITLE
graph query v2.

### PR DESCRIPTION
**Pull request by scarlett950810**
Changes:
1.change popup from click to doubleClick. (now click to show children and doubleClick to get popup window allowing users to select and unselect nodes).
2.for the main canvas, reduced stabilization time.
3.improved animation physics.
4.change canvas and right bar height the same as windows height.
5.solved the issue of delay of change of color.

### CODE REVIEW

**Comment by shubhamgoyal** [comment]
@scarlett950810 you really need to apply more time to giving titles to your pull requests. The title should be a concise summary of what the pull request is about. A third person should be able to look at the title and get an idea of what the pull request is about.

File: site/trendspedia/static/vis/helper.js [hunk 1]
```diff
@@ -372,14 +390,15 @@ var removeReminderForDeleteAllButton = function(id){
 
     deleteReminder(id);
     // delete from selectedNodesArray
-    deleteSelectedNode_fromArray(id)
+    deleteSelectedNode_fromArray(id);
```
Comment (allwinpb) [inline site/trendspedia/static/vis/helper.js]: Hey try to stick to one style snake_case vs camelCase. JS is usually done in camelCase.

**Comment by allwinpb** [comment]
Also regarding markdown, leaving two spaces at the end of a line automatically inserts a line break, so the description would look proper.

**Comment by scarlett950810** [comment]
For the naming style, snake_case functions names are used for code readability, i.e. snake_case are helper functions used in corresponding camelCase.
e.g. deleteSelectedNode_fromArray are called in deleteSelectedNode

**Pull request merged**